reference/sectionC_interpretation.inc

```diff
@@ -23,6 +23,8 @@ Interpretation of imports
 Units reference
 ---------------
 
+A "units reference" refers to the :code:`units_ref` attribute in :code:`import units` items, and the :code:`units` attribute used in :code:`unit` and :code:`variable` items.
```

Comment:
- Why the `"` around "units reference"? I can see that it's also done in C4, but should we really do it? (Just asking as I think I might be fine with it.)
- Wouldn't it be more accurate to have " ... refers to **the value of** the :code:`units_ref` attribute... and **the value of** the :code:`units` attribute..."? (If so, we ought to update C4 too.)
- Technically speaking, we don't `import units` items, just `import` items for units. (Same for C4.)

Reply:
1. Because I got really confused my first time reading this section.  I didn't know what the phrase "units reference" meant - this is supposed to define it, hence the ".  If there's a clearer way am happy to change it?
2,3. You're right, I think ... will update.

Reply:
Done in cff1dc9